src/entitysdk/models/agent.py

```diff
@@ -2,26 +2,51 @@
 
 from typing import Literal
 
+from pydantic import Field
+
 from entitysdk.models.core import Identifiable
 
 
 class Agent(Identifiable):
     """Agent model."""
 
-    type: str
-    pref_label: str
+    type: str = Field(
+        ...,
+        description="The type of this agent.",
+    )
+    pref_label: str = Field(
+        ...,
+        description="The preferred label of the agent.",
+    )
```

Comment:
Can we use the Annotated syntax, that seems more recommended for new code in both fastapi and pydantic?

Reply:
Sure, done.